libs/blocks/global-footer/global-footer.css

```diff
@@ -228,6 +222,7 @@
   column-gap: 5px;
   flex-wrap: wrap;
   margin: 0;
+  font-size: 11px;
```

Comment:
Why is the font size different for the copyright section?

Reply:
This was a change specified in the mockups, but we can revert this if its not needed.
<img width="652" alt="Screenshot 2024-08-06 at 10 12 36 AM" src="https://github.com/user-attachments/assets/20a8559b-ecb5-4b81-a5fb-85f2214c1902">

Reply:
@saashapina - is the other content in the footer 11px as well?

Reply:
No, it looks like the other content in the footer remains the same as the original at 12px. It was only the details in this privacy section that was updated to the 11px.

Reply:
I did some investigations and looks like the copyright text was 11px in the AEM Global Navigation, meaning the change is correct. I will mark this as resolved, no further action needed. Thanks!